06_gpu_and_ml/llm-structured/instructor/inference.py

```diff
@@ -0,0 +1,77 @@
+# # Fast inference with vLLM (Mistral 7B)
```

Comment:
Am I missing something here? Looks like this is demonstrating vLLM inference as an openAI-compatible server, not anything to do with `instructor`?

Also we've got a lot of vLLM copy-pasta going on, I'm making updates in one place while we're spawning copies of the old version elsewhere :D. We need a better approach here. (Hard problem though!)